mpas_analysis/shared/climatology/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .climatology import get_lat_lon_comarison_descriptor, get_remapper, \
+    get_mpas_climatology_file_names, get_observation_climatology_file_names, \
+    compute_monthly_climatology, compute_climatology, cache_climatologies, \
+    update_start_end_year, add_years_months_days_in_month, \
+    remap_and_write_climatology
```

Comment:
We should probably update other analysis tasks at some point to use the new, less redundant imports.  But there is nothing wrong with keeping the old way for now. 

One reason I'm not in a big hurry is that a lot will get updated when I make analysis tasks classes in #128, or when I address #196.